openml/testing.py

```diff
@@ -104,9 +114,43 @@ def tearDown(self):
         openml.config.server = self.production_server
         openml.config.connection_n_retries = self.connection_n_retries
 
+    @classmethod
+    def _track_test_server_dumps(self, entity_type, entity_id):
```

Comment:
Could you please rename that function to something like `mark_entity_for_removal`?